src/firefly/js/ui/panel/TabPanel.jsx

```diff
@@ -194,15 +194,18 @@ export class Tab extends Component {
         const textStyle = maxTitleWidth ? {float: 'left', width: maxTitleWidth-(removable?14:0)} : {};
 
         return (
-            <li className={tabClassName} onClick={() => onSelect(id,name)}>
+            <li className={tabClassName} onClick={() => !this.removed && onSelect(id,name)}>
```

Comment:
Ah, clicking tab remove button triggered tab selection! 

I wonder why it did not create issues for more than 2 tables.